server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/DeletedMessagesVaultTest.java

```diff
@@ -114,12 +165,17 @@ public void setup() throws Throwable {
         homerAccessToken = authenticateJamesUser(baseUri(jmapServer), HOMER, PASSWORD);
         bartAccessToken = authenticateJamesUser(baseUri(jmapServer), BART, BOB_PASSWORD);
 
-        webAdminApi = WebAdminUtils.spec(jmapServer.getProbe(WebAdminGuiceProbe.class).getWebAdminPort());
+        WebAdminGuiceProbe webAdminGuiceProbe = jmapServer.getProbe(WebAdminGuiceProbe.class);
+        webAdminApi = given()
+            .spec(WebAdminUtils
+                .buildRequestSpecification(webAdminGuiceProbe.getWebAdminPort())
+                .build());
     }
 
     @After
-    public void tearDown() {
+    public void tearDown() throws Exception {
         jmapServer.stop();
+        FileUtils.forceDelete(fileSystem.getBasedir());
```

Comment:
Can we : 
 - Move `org.apache.james.blob.export.file.FileSystemExtension` into server-core test jar?
 - Use this extention here

Reply:
I can move it, but currently, DeletedMessagesVaultTest is using Junit 4, It will take effort to migrate this class to Junit 5

Reply:
Crap, good point. That's OK for now

Reply:
you can use TemporaryFolder from junit 4 rules